packages/visualizations/addon/templates/components/navi-visualizations/table.hbs

```diff
@@ -5,13 +5,14 @@
 </div>
 <div class="table-widget__horizontal-scroll-container">
   <div class= 'table-widget-container'>
-      {{#if isEditing}}
+      {{#if (and (feature-flag 'enableTableEditing') isEditing)}}
         <div class='table-header-row'>
           {{#each columns as |column|}}
             <div class='table-header-cell {{column.type}}'>
               {{input
                 class='table-header-cell__input'
-                value=(readonly column.displayName)
+                value=column.displayName
```

Comment:
Can we keep the `readonly`?

Reply:
What is the reason to keep it readonly? If it is readonly, we need to manually update displayName.

Reply:
Two way data binding has caused us problems in the past.
https://dockyard.com/blog/2015/10/14/best-practices-data-down-actions-up